### ISSUE TITLE
contenthash generates same filename for different contents

### ISSUE THREAD

**Issue by zlk89** [webpack-contrib/mini-css-extract-plugin#454]
- Operating System: macOS 10.14.6
- Node Version: v8.11.2
- NPM Version: 6.0.1
- webpack Version: 4.41.2
- mini-css-extract-plugin Version: 0.8.0

### Expected Behavior

Should generate different filename when content is changed.

### Actual Behavior

Same filename was generated when content is updated.

### How Do We Reproduce?

https://github.com/zlk89/webpack-css-contenthash-diff
- `yarn` to install packages
- `yarn build`
- under `dist` folder, you will find two css files with same content hash, while content is different.

Basically contenthash is same, when import statement order is different:
```
// in js
@import 'a.scss';
@import 'b.scss';
```

```
// in js
@import 'b.scss';
@import 'a.scss';
```

**Comment by evilebottnawi** [comment]
Please create reproducible test repo

**Comment by zlk89** [comment]
@evilebottnawi updated description with a minimal reproducible repo and steps

**Comment by evilebottnawi** [comment]
Send a fix, thanks for the PR

**Comment by zlk89** [comment]
@evilebottnawi any update? We are getting this issue again

**Comment by evilebottnawi** [comment]
@zlk89 no right now, but will be fixed in near future

**Comment by zlk89** [comment]
@alexander-akait looks like the issue is still existed. I have bumped the repo to latest webpack (v5.10) and mini css extract plugin (v1.3.2)

**Comment by alexander-akait** [comment]
@zlk89 Yep, a lot of work, but it in my TODO, sorry :disappointed:

**Comment by zlk89** [comment]
^ bump, this bug essentially prevent us from utilizing long term cache on all css generated by this plugin

**Comment by zlk89** [comment]
verified the bug has been fixed
